connection/connection.go

```diff
@@ -45,16 +46,17 @@ func Dial(url string, ops ...Option) (*Redialer, error) {
 // Option to configure Reconnector.
 type Option func(r *Redialer)
 
-// Channel opens new amqp.Channel.
-// Will create connection if channel is closed.
-func (r *Redialer) Channel() (*amqp.Channel, error) {
+// Channel opens new amqp.Channel wit re-open capabilities.
```

Comment:
typo, should be `Channel opens new amqp.Channel wit re-open capabilities.`

Reply:
updated